src/patternfly/components/Masthead/masthead.scss

```diff
@@ -3,174 +3,108 @@ $pf-v5-c-masthead--breakpoint-map: build-breakpoint-map();
-  --#{$masthead}--c-context-selector--m-full-height__toggle--BorderTopColor: transparent;
-  --#{$masthead}--c-context-selector--m-full-height__toggle--BorderBottomColor: transparent;
-
-  // Dropdown
-  --#{$masthead}--c-dropdown__toggle--before--BorderTopColor: var(--#{$masthead}--item-border-color--base);
-  --#{$masthead}--c-dropdown__toggle--before--BorderRightColor: var(--#{$masthead}--item-border-color--base);
-  --#{$masthead}--c-dropdown__toggle--before--BorderLeftColor: var(--#{$masthead}--item-border-color--base);
-  --#{$masthead}--c-dropdown--m-full-height__toggle--before--BorderTopColor: transparent;
-  --#{$masthead}--c-dropdown--m-full-height__toggle--before--BorderBottomColor: transparent;
-
-  // Menu toggle
-  --#{$masthead}--c-menu-toggle--before--BorderTopColor: var(--#{$masthead}--item-border-color--base);
-  --#{$masthead}--c-menu-toggle--before--BorderRightColor: var(--#{$masthead}--item-border-color--base);
-  --#{$masthead}--c-menu-toggle--before--BorderLeftColor: var(--#{$masthead}--item-border-color--base);
+  --#{$masthead}--m-display-inline__toggle--GridColumn: 1;
 
-  // Toolbar
-  --#{$masthead}--c-toolbar--BackgroundColor: var(--#{$masthead}--BackgroundColor);
-  --#{$masthead}--c-toolbar--AlignItems--base: center;
-  --#{$masthead}--c-toolbar__content--PaddingRight: 0;
-  --#{$masthead}--c-toolbar__content--PaddingLeft: 0;
-  --#{$masthead}--c-toolbar__expandable-content--PaddingTop: var(--pf-t--global--spacer--md);
-  --#{$masthead}--c-toolbar__expandable-content--PaddingRight: var(--#{$masthead}--inset);
-  --#{$masthead}--c-toolbar__expandable-content--PaddingBottom: var(--pf-t--global--spacer--md);
-  --#{$masthead}--c-toolbar__expandable-content--PaddingLeft: var(--#{$masthead}--inset);
-  --#{$masthead}--c-toolbar__expandable-content--before--BorderTopWidth: var(--pf-t--global--border--width--divider--default);
-  --#{$masthead}--c-toolbar__expandable-content--before--BorderTopColor: var(--#{$masthead}--item-border-color--base);
+  // Masthead toolbar
+  --#{$masthead}--c-toolbar__expandable-content--BoxShadow: var(--pf-v5-global--BoxShadow--md-bottom); // update after toolbar updates
+  --#{$masthead}--c-toolbar__expandable-content--BorderBlockStart: var(--#{$masthead}--BorderWidth) solid var(--#{$masthead}--BorderColor);
 }
 
+// Masthead display stack
 @mixin pf-v5-c-masthead--m-display-stack {
   --#{$masthead}--GridTemplateColumns: var(--#{$masthead}--m-display-stack--GridTemplateColumns);
+  --#{$masthead}--ColumnGap: var(--#{$masthead}--m-display-stack--ColumnGap);
   --#{$masthead}__main--GridColumn: var(--#{$masthead}--m-display-stack__main--GridColumn);
-  --#{$masthead}__main--MinHeight: var(--#{$masthead}--m-display-stack__main--MinHeight);
-  --#{$masthead}__main--Order: var(--#{$masthead}--m-display-stack__main--Order);
   --#{$masthead}__main--FlexBasis: var(--#{$masthead}--m-display-stack__main--FlexBasis);
```

Comment:
Doesn't look like this var is being used anymore (also on line ~83 inside the mixin)